### DISCUSSION TITLE
Make user custom properties added visible in admin dashboard under security> User

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#15032] [Q&A]
I write this code to save User and make  custom  properties visible in admin dashboard under security> User
but I need to know the fields to create into this two Classes UserCustomFields and UserCustomFieldsPart in my orchard Core solution
![2024-01-05_17-23-23](https://github.com/OrchardCMS/OrchardCore/assets/154245181/b9dffb6f-909e-41a0-a44b-edde76b47bef)

**Comment by lampersky** [comment]
@YoumnaJ just create this missing class:

```
    public class UserCustomFields : ContentPart
    {
        public UserCustomFields() { }
        
        public TextField Name { get; set; }
    }
```

don't forget to register your part in Startup.cs:

```
public class Startup : StartupBase
{
    public override void ConfigureServices(IServiceCollection services)
    {
        services.AddContentPart<UserCustomFields>();
    }
}
```

**Reply by lampersky** [reply]
I'd suggest you to watch this [video](https://www.youtube.com/watch?v=bayT58i7DVY)
